client/src/core/client/admin/components/ModerationReason/Reasons.tsx

```diff
@@ -0,0 +1,102 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+import { GQLREJECTION_REASON_CODE } from "coral-framework/schema";
+import { /* Button, */ RadioButton } from "coral-ui/components/v2";
+import { Button } from "coral-ui/components/v3";
+
+import { unsnake } from "./formatting";
+
+import styles from "./ModerationReason.css";
+
+export interface Props {
+  onCode: (code: GQLREJECTION_REASON_CODE) => void;
+  selected: GQLREJECTION_REASON_CODE | null;
+  onAddExplanation: () => void;
+}
+
+const AddExpanationButton: FunctionComponent<{ onClick: () => void }> = ({
```

Comment:
Can you update this component name to address the typo...`AddExpanationButton` --> `AddExplanationButton`?

Reply:
✅